app/forms/draft_work_form.rb

```diff
@@ -20,11 +20,15 @@ class DraftWorkForm < Reform::Form
   property :access
   property :license
   property :agree_to_terms
+  property :created_type, virtual: true, prepopulator: (proc do |*|
```

Comment:
This is the pattern to set a virtual property based on a currently set value: using a reform prepopulator. The parens around the proc are to prevent a syntax error.

**NOTE**: these prepopulators do not automatically fire. They must be invoked in the form via `form.prepopulate!`. This is done in the controller, which is why testing this in specs that do not go through controller code must call it manually.